src/react/components/pages/editorPage/editorSideBar.tsx

```diff
@@ -78,13 +78,13 @@ export default class EditorSideBar extends React.Component<IEditorSideBarProps,
         this.setState({
             selectedAsset: asset,
         }, () => {
-            this.props.onAssetSelected(asset);
             this.listRef.current.forceUpdateGrid();
         });
     }
 
     private onAssetClicked(asset: IAsset) {
         this.selectAsset(asset);
+        this.props.onAssetSelected(asset);
```

Comment:
why are we moving this here?

Reply:
During testing a encountered an infinite style look in some conditions of the editor page.  The `EditorSideBar` exposes an `onAssetSelected` prop as well as a `selectedAsset` prop which the editor bind to.  During the page lifecycle, the editor loads the assets for the properties.  Sets the first asset as `selectedAsset`.  When the user clicks a new image in the sidebar that triggers `onAssetSelected` event handler to fire which was then causing the editor to set its internal selected asset state, which in turn is then rebound to the sidebar. 

The fix is to only have the sidebar fire its `onAssetSelected` event handler during a user interaction flow that changes the selected asset, not via property binding - this is more consistent with how event handler work in general within web apps.